docs/user-guide/catalog.md

```diff
@@ -218,6 +225,11 @@ Enabling that option, all layers added to the map from this catalog source will
 
 * *Layer tile size*: it represents tile size (width and height) to be used for tiles of all layers added to the map from the catalog source (`256x256` or `512x512`). For layers already loaded on the map, it is possible to change the tile size through the [Layer Settings](https://mapstore.readthedocs.io/en/latest/user-guide/layer-settings/#display) tool as usual.
 
+* *Server Type*: to choose the type of server from which the url of the entered CSW service comes. The options are two: `Geoserver` or `No Vendor`which can be mapproxy, mapserver or other.
+
+!!! note
+    If set on **No Vendor** then *TILED* query parameter, for all layers added to the map from this source, is not sent to the remote wms server, and the `tiled` and `localizedLayerStyles` options are hidden in the layer parameters.
```

Comment:
If the No Vendor is set, then MS will not use any vendor option supported only by GeoServer in the OGC requests where this source is involved.